.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
Configured vscode to use the local version of typescript, so that error messages in the editor match `yarn types`. Not sure why this isn't the default.